src/utils/index.ts

```diff
@@ -52,7 +52,9 @@ export const validateOptions = (options?: Options): ValidatedOptions => {
   return {
     customBackend: options.customBackend,
     projectId: options.projectId,
-    isTestnet: options.isTestnet,
+    isTestnet:
+      options.isTestnet ||
```

Comment:
`??` would be less intrusive. otherwise devs with old token, who are not using `isTestnet` option at all (because they always wanted mainnet), will get unnecessary warning